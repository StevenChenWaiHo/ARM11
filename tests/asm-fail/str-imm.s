str r3,=10