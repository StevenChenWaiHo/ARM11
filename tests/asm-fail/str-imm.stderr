tests/asm-fail/str-imm.s:1:11: Cannot use str with immediate number
